README.md

```diff
@@ -21,16 +21,16 @@ The Distributed Generation Market Demand (dGen) Model
 ## Get Your Tools
 Install Docker for [(Mac)](https://docs.docker.com/docker-for-mac/install/) or [(Windows)](https://docs.docker.com/docker-for-windows/install/)
 
-- Important: In Docker, go into Docker > Preferences > Resources and up the allocation for disk size image for Docker. 16 GB is recommended for smaller (state level) databasese. 32 GB is recommended for ISO specific databases. 70+GB is required for restoring the national level database. If you get a memory issue then you'll need to up the memory allocation and or will need to prune past failed images/volumes. Running the below docker commands will clear these out and let you start fresh:
+- Important: In Docker, go into Docker > Settings ("Gear" icon in top-right corner of application) > Resources and up the allocation for disk size image for Docker. 16 GB is recommended for smaller (state level) databasese. 32 GB is recommended for ISO specific databases. 70+GB is required for restoring the national level database. If you get a memory issue then you'll need to up the memory allocation and or will need to prune past failed images/volumes. Running the below docker commands will clear these out and let you start fresh:
 ```
    $ docker system prune -a 
    $ docker volume prune -f
 ``` 
 - Please refer to Docker’s [documentation](https://docs.docker.com/reference/) for more details.
 
-- Install [Anaconda for Python 3.7](https://www.anaconda.com/distribution/). Users with VPNs may need to turn their VPNs off while installing or updating Anaconda.
+- Install [Anaconda for Python 3.7](https://www.anaconda.com/download/success). Users with VPNs may need to turn their VPNs off while installing or updating Anaconda.
 
-- Install [PgAdmin](https://www.pgadmin.org/download/). Ignore all of the options for docker, python, os host, etc.
+- Install [PgAdmin](https://www.pgadmin.org/download/). Download pgAdmin 4 based on your operating system (i.e., macOS, Windows, etc.).
```

Comment:
```suggestion
- Install [PgAdmin](https://www.pgadmin.org/download/). Download pgAdmin 4 based on your operating system (i.e., macOS, Windows, etc.). If prompted, ignore all of the options for installing docker, python, os host, etc.
```
Just keeping this note in case user is prompted during installation